tests/Unit/HealthCheckerTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+namespace Mongolid\Laravel;
+
+use Mockery as m;
+use MongoDB\Client;
+use MongoDB\Driver\Exception\ConnectionTimeoutException;
+use MongoDB\Driver\Manager;
+use Mongolid\Connection\Connection;
+
+class HealthCheckerTest extends TestCase
+{
+    public function testHealthOK(): void
+    {
+        // Set
+        $connection = m::mock(Connection::class);
+        $client = m::mock(Client::class);
+        $manager = m::mock(new Manager('mongodb://foo'));
+        $checker = new HealthChecker($connection);
+
+        // Expectations
+        $connection->shouldReceive('getClient')
+            ->andReturn($client);
+
+        $client->shouldReceive('getManager')
+            ->andReturn($manager);
+
+        $manager->shouldReceive('executeCommand')
+            ->andReturn([
+                'ok' => 1,
+            ]);
```

Comment:
(and `use MongoDB\Driver\Command; `)
```suggestion
        $manager->expects()        
            ->executeCommand('defaultTestDatabase', m::type(Command::class))
            ->andReturn([
                'ok' => 1,
            ]);
```